queries/lockup.js

```diff
@@ -0,0 +1,59 @@
+const pageResults = require('graph-results-pager');
+
+const ws = require('isomorphic-ws');
+const { SubscriptionClient } = require('subscriptions-transport-ws'); 
+
+const { request, gql } = require('graphql-request');
+
+const { graphAPIEndpoints, graphWSEndpoints } = require('./../constants')
+const { timestampToBlock } = require('./../utils')
+
+module.exports = {
+    async users({block = undefined, timestamp = undefined, user_address = undefined} = {}) {
```

Comment:
Should this be `user` instead of `users`?

Also this query is returning an error for me:
`sushiData.lockup.users({user_address: '0x4bb4c1B0745ef7B4642fEECcd0740deC417ca0a0'}).then(info => console.log(info))`

Reply:
Yup, you're right, fixed